cli-toolchain/graphql.md

```diff
@@ -1874,215 +2166,740 @@ type Comment {
 }
 ```
 
-#### Generates
+#### Generates
+
+In order to keep connection queries fast and efficient, the GraphQL transform manages
+global secondary indexes (GSIs) on the generated tables on your behalf. In the future we
+are investigating using adjacency lists along side GSIs for different use cases that are
+connection heavy.
+
+> **Note** The `@connection` directive manages these GSIs under the hood but there are limitations to be aware of. After you have pushed a `@connection` directive you should not try to change it. If you try to change it, the DynamoDB UpdateTable operation will fail due to one of a set of service limitations around changing GSIs. Should you need to change a `@connection`, you should add a new `@connection` that implements the new access pattern, update your application to use the new `@connection`, and then delete the old `@connection` when it's no longer needed.
```

Comment:
We should reword this. First "under the hood" isn't professional. Second this talks about limits in the CloudFormation of DynamoDB in a negative light, but those controls exist for a reason. We can remove those lines and just speak factually about what customers need to do.